polyfill/lib/date.mjs

```diff
@@ -8,12 +8,12 @@ import {
   ISO_YEAR,
   ISO_MONTH,
   ISO_DAY,
-  HOUR,
-  MINUTE,
-  SECOND,
-  MILLISECOND,
-  MICROSECOND,
-  NANOSECOND,
+  ISO_HOUR,
+  ISO_MINUTE,
+  ISO_SECOND,
```

Comment:
For seconds and smaller units, do they need to be calendar-aware?  Are there any use cases we know about that require calendar-specific measurement of seconds or nanoseconds? Or are we just making them calendar-aware for consistency?

Reply:
I think we should probably rename them to ISO in any case, but yes it does seem that there should not be use cases where 1000 milliseconds ≠ 1 second.